app/default/helpers.py

```diff
@@ -0,0 +1,16 @@
+import requests
+from app.config import FORMS_SERVICE_PUBLIC_HOST
+
+
+def get_return_to_runner_token(form_name: str, formatted_application_payload):
```

Comment:
maybe this could be get_return_to application_token? (runner could be an ambiguous term)

Reply:
another good rename suggestion 🥇